util/src/main/java/io/kubernetes/client/util/ClientBuilder.java

```diff
@@ -100,28 +99,46 @@ public static ClientBuilder standard() throws IOException {
     return standard(true);
   }
 
+  public static ClientBuilder context(String context) throws IOException {
+    final File kubeConfig = getKubeConfigFile();
+    ClientBuilder clientBuilder = getClientBuilder(true, kubeConfig, context);
```

Comment:
Don't change the signature here, just call `kubeconfig.setContext(context)` before you call `getClientBuilder()`

Reply:
`kubeConfig` is of type `File` , `setContext(context)` can only be set at a later stage when building the client